addons/website/__manifest__.py

```diff
@@ -101,4 +100,334 @@
     'application': True,
     'post_init_hook': 'post_init_hook',
     'uninstall_hook': 'uninstall_hook',
+    'assets': {
```

Comment:
Perhaps you already planned it but just to make sure: don't forget to convert the `qweb` manifest key to an asset bundle eventually